### PR TITLE
Merging 2dotfangle - analyzing the impact of defocus on the MTF

### PR DESCRIPTION
**Pull request by wandell**
Trisha and I were  experimenting to understand how defocus impacts the MTF.  We wrote a tutorial, t_wvfMTF because some of the results surprised us.  

Specifically, as we defocus for a diffraction-limited optics a defocus of 0.15 diopters has only a small impact on the MTF.  But when we run the same code using the human optics ('wvf human') the same defocus has a very different and larger effect.  So, a specific amount of defocus does not have the same impact on the MTF for different base Z polynomials.  Maybe we shouldn't have been surprised.

The t_wvfMTF illustrates this effect.

While making the code run, we found some issues that we had to fix up in wvfGet.  And we adjusted wvfPlot.

The ISETBio validations appear to still run correctly.  

In summary, the main changes are

- I have opticsCreate() return the wavefront struct for the human wavefront case.
- We fixed an issue with wvfGet and 'samples spacing'.  That code was not handling distance (meters) and angle (deg) requests correctly.
- We added the t_wvfMTF function.
- We edited the wvfPlot function a bit.

### CODE REVIEW

**Comment by DavidBrainard** [comment]
This is interesting.  I think it relates to something else that has been on the back burner for a while, namely that the effects of chromatic aberration can be quite complex because of the same sort of interaction.
